metricbeat/docs/fields.asciidoc

```diff
@@ -1579,6 +1578,166 @@ type: long
 
 --
 
+*`aws.billing.Currency`*::
++
+--
+Estimated charges currency unit.
+
+type: keyword
+
+--
+
+*`aws.billing.ServiceName`*::
++
+--
+Service name for the maximum estimated charges.
+
+type: keyword
+
+--
+
+
+*`aws.billing.AmortizedCost.amount`*::
++
+--
+Amortized cost amount
+
+type: double
+
+--
+
+*`aws.billing.AmortizedCost.unit`*::
++
+--
+Amortized cost unit
+
+type: keyword
+
+--
+
+
+*`aws.billing.BlendedCost.amount`*::
++
+--
+Blended cost amount
+
+type: double
+
+--
+
+*`aws.billing.BlendedCost.unit`*::
++
+--
+Blended cost unit
+
+type: keyword
+
+--
+
+
+*`aws.billing.NormalizedUsageAmount.amount`*::
++
+--
+Normalized usage amount
+
+type: double
+
+--
+
+*`aws.billing.NormalizedUsageAmount.unit`*::
++
+--
+Normalized usage amount unit
+
+type: keyword
+
+--
+
+
+*`aws.billing.UnblendedCost.amount`*::
++
+--
+Unblended cost amount
+
+type: double
+
+--
+
+*`aws.billing.UnblendedCost.unit`*::
```

Comment:
It might be common with AWS module, but are we sure we want the naming to be in camelcase format?

Reply:
Yeah we want to keep the metric names straight out from CloudWatch for AWS module 😄 At least right now, this is the plan.

Reply:
👍